src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 2;
```

Comment:
The constant `SHORT_TERM` should be named to reflect the threshold for a discount, such as `MIN_DAYS_FOR_DISCOUNT`, and its value should be `3` to match the task description that states a discount is applied for 3 or more days.